test/unit/task_executor_test.go

```diff
@@ -30,6 +30,7 @@ func TestGroupTxExecutorEmitAndTransfer(t *testing.T) {
 	require.Empty(t, initMsg)
 
 	user1 := ledger.NewWallet()
+	require.NoError(t, user1.AddUser(base58.Encode(user1.PublicKeyEd25519), "user1"))
```

Comment:
You don't need to create an AddUser
You need to make a change in NewWallet
NewWalet(optioanal ....)
NewWalet(WithUserID(“user1”))